src/app/component/CookiePopup/CookiePopup.component.js

```diff
@@ -0,0 +1,55 @@
+/**
+* ScandiPWA - Progressive Web App for Magento
+*
+* Copyright © Scandiweb, Inc. All rights reserved.
+* See LICENSE for license details.
+*
+* @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+* @package scandipwa/base-theme
+* @link https://github.com/scandipwa/base-theme
+*/
+
+import { PureComponent } from 'react';
+
+import BrowserCookie from 'Util/BrowserCookie';
+import Link from 'Component/Link';
+
+import './CookiePopup.style';
+
+export const COOKIE_POPUP = 'cookie_popup';
+
+class CookiePopup extends PureComponent {
+    state = { isAccepted: BrowserCookie.getItem(COOKIE_POPUP) || false };
```

Comment:
Are you sure we would like to store information about what user has selected in popup in cookies? Maybe we can use local-storage? What is the reason?